packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1183,6 +1183,7 @@ class ThemeData with Diagnosticable {
   ///   * FAB: [FloatingActionButton]
   ///   * Extended FAB: [FloatingActionButton.extended]
   ///   * Cards: [Card]
+  ///   * TextFields: [TextField]
```

Comment:
Would be helpful to mention InputDecorator

Reply:
Added